meson.build

```diff
@@ -22,15 +23,19 @@ import('pkgconfig').generate(
 	name: meson.project_name(),
 	description: 'JIT assembler for x86(IA32), x64(AMD64, x86-64)',
 	version: meson.project_version(),
-	url: 'https://github.com/herumi/xbyak'
+	url: 'https://github.com/herumi/xbyak',
+	variables: ['includedir=@0@'.format(include_dir)],
+	dataonly: true,
```

Comment:
According to [Meson's documentation](https://mesonbuild.com/Pkgconfig-module.html), this is not allowed:

> The names `prefix`, `libdir` and `includedir` are reserved and may not be used.

You probably added this to work around the fact that `dataonly: true` generates a .pc file without an `includedir` variable, but even then this is not ideal for two reasons:

- No `prefix` variable gets defined, which can cause issues since it is often overridden by users
- No `Cflags: -I${includedir}` directive gets written to the file, meaning that compilation will fail to work when the library is installed outside the default include directories since `includedir` is unused.
- `dataonly` is only available with Meson `0.54.0` or newer, and this project does not require it.

The solution to this is simply to leave out the `variables` and `dataonly` keyword arguments, and to set `install_dir` to `get_option('datadir')/'pkgconfig'`. Since no libraries are passed to the `pkgconfig.generate()` call, `libdir` and `Libs:` do not get written to the file, making it suitable for `/usr/share`.